### PR TITLE
Extra start and stop values

### PR DESCRIPTION
**Pull request by PhilPotter**
Introduce inlet export and import values, and serialisation/deserialisation code, as well as a utility function
for easier testing from the server API project.

### CODE REVIEW

File: ocpp-v1_6/src/main/java/eu/chargetime/ocpp/feature/profile/ClientCoreProfile.java [hunk 1]
```diff
@@ -214,6 +214,31 @@ public StartTransactionRequest createStartTransactionRequest(
...
+   * @param idTag required. Authorization identification tag.
+   * @param meterStart required. The initial value of the meter.
+   * @param inletExportStart not required. The initial value of inlet exported Wh.
+   * @param inletImportStart not required. The initial value of inlet imported Wh.
+   * @param timestamp required. Time of start.
+   * @return an instance of {@link StartTransactionRequest}.
+   * @see StartTransactionRequest
+   * @see StartTransactionFeature
+   */
+  public StartTransactionRequest createSolarStartTransactionRequest(
```
Comment (Amxt) [inline ocpp-v1_6/src/main/java/eu/chargetime/ocpp/feature/profile/ClientCoreProfile.java]: this is not specifically for solar, we're reading values from the inlet, there should be no reference to solar

Comment (PhilPotter) [reply ocpp-v1_6/src/main/java/eu/chargetime/ocpp/feature/profile/ClientCoreProfile.java]: Renamed, thanks.

**Review by DanielVocke — CHANGES_REQUESTED** [review]
why did you commit the imp file changes? Otherwise looks good.

**Review by Amxt — APPROVED** [review]

**Review by Amxt — APPROVED** [review]

**Comment by PhilPotter** [comment]
Approved by Dan V verbally, although we will need to remove this code ultimately, due to third-party compatibility reasons.

**Pull request merged**